### PR TITLE
update wc-rpc-methods.md

### PR DESCRIPTION
**Pull request by pedrouid**

### CODE REVIEW

**Comment by xzilja** [comment]
Me gusta 👍

File: wc-rpc-methods.md [hunk 1]
```diff
@@ -123,35 +92,39 @@ Used to settle a session over topic B.
...
-
-### wc_sessionExtend
-
+### wc_sessionUpdateExpiry
 Used to extend the lifetime of a session.
-
 - The expiry is the absolute timestamp of the expiration date, in seconds.
-
 ```jsonc
 // wc_sessionExtend params
```
Comment (jakubuid) [inline wc-rpc-methods.md]:
```suggestion
// wc_sessionUpdateExpiry params
```

**Comment by xzilja** [comment]
Can we also flatten `wc_sessionRequest` and `wc_sessionEvent` methods? i.e.

```
// wc_sessionRequest params
{
    method: "wc_sessionRequest";
    params: {
      method: string;
      params: unknown;
      chainId: string;
    };
}
```

and 

```
// wc_sessionEvent params
{
  method: "wc_sessionEvent";
  params: {
    name: string;
    data: unknown;
    chainId: string;
}
```

**Comment by xzilja** [comment]
@pedrouid is same change (https://github.com/WalletConnect/sdk-team/pull/27/files/502eb7d034002fa5be0b20d2af335d325c03f69b..41455ec62099f499d1221b2d9b59d05170705bfb) needed in `wc_sessionDelete` ?

**Comment by pedrouid** [comment]
> @pedrouid is same change (https://github.com/WalletConnect/sdk-team/pull/27/files/502eb7d034002fa5be0b20d2af335d325c03f69b..41455ec62099f499d1221b2d9b59d05170705bfb) needed in `wc_sessionDelete` ?

It should be

**Pull request merged**